languages/ruby/lib/ruby-bitwarden-sdk.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'dry-types'
+
+require_relative 'schemas'
+require_relative 'extended_schemas/schemas'
+require_relative 'command_runner'
+require_relative 'bitwarden_lib'
+require_relative 'projects'
+require_relative 'secrets'
+
+class BitwardenClient
+  attr_reader :bitwarden, :project_client, :secrets_client
+
+  def initialize
+    api_url = ENV['BITWARDEN_API_URL'] || 'https://api.bitwarden.com'
+    device_type = ENV['BITWARDEN_DEVICE_TYPE'] || 'MacOsDesktop'
+    identity_url = ENV['BITWARDEN_IDENTITY_URL'] || 'https://identity.bitwarden.com'
+    user_agent = ENV['BITWARDEN_USER_AGENT'] || 'SDK'
+
+    client_settings = ClientSettings.new(
+      api_url: api_url,
+      device_type: device_type,
+      identity_url: identity_url,
+      user_agent: user_agent
+    )
```

Comment:
The SDK should handle nullable values and we should rely on the values in the rust code instead of duplicating them here.

Reply:
these are handled now